CHANGEPLAN.md

```diff
@@ -1,9 +1,64 @@
 # Governance Evolution
 
-## Stages
+## Approach
 
-### Stage 0
+The general approach to transitioning from the existing Cloud Foundry
+Foundation technical governance process to the new process is to start
+with the TOC and then allow that new TOC to work through the details
```

Comment:
Maybe we can expand TOC for the first instance in this doc

Reply:
Updated to do this